app/modules/settings/reducers/settings.js

```diff
@@ -1,13 +1,16 @@
-import { SETTINGS_CHANGE_CLONE_DESTINATION } from "../constants"
+import { SETTINGS_CHANGE_CLONE_DESTINATION, SETTINGS_SET_USERNAME } from "../constants"
 
 const initialState = {
-  cloneDestination: "/tmp"
+  cloneDestination: "/tmp",
```

Comment:
:art: add a `// TODO` here to use [`os.tmpdir()`](https://nodejs.org/dist/latest-v8.x/docs/api/os.html#os_os_tmpdir) rather than this hard-coding.

Reply:
Makes sense, just added that!